src/content/blog/experimental-server-side-rendering/index.mdx

```diff
@@ -8,6 +8,8 @@ authors:
   - matthew
 ---
 
+import netlifyDeployVideo from "./netlify-deploy-2.mp4"
```

Comment:
@jasikpark Sorry, I totally missed this PR comment after letting the PR go stale during launch week!

Yep that's totally possible, at that time I had just missed the leading `_` so that the file is hidden from the content collections API

Importing a `.mp4` file ends up returning the hashed file path, basically `*.mp4?url` without needing the query param since it wouldn't make much sense for the import to return the raw binary content or anything like that